test/tls_test.go

```diff
@@ -2,21 +2,23 @@ package test
 
 import (
 	"crypto/tls"
+	"fmt"
 	"testing"
 
 	"github.com/miekg/dns"
 )
 
 func TestDNSoverTLS(t *testing.T) {
```

Comment:
Can you rename this test to `TestTLS`, and make it execute two test cases: the existing unaltered dot case, and the new dot+doh case?